templates/tc476356/application-template-dockerbuild.json

```diff
@@ -0,0 +1,382 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "ruby-helloworld-sample",
+    "creationTimestamp": null,
+    "annotations": {
+      "description": "This example shows how to create a simple ruby application in openshift origin v3"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
+      "apiVersion": "v1beta3",
```

Comment:
I copied the latest origin template files from https://github.com/openshift/origin/tree/master/examples/sample-app yesterday.